resources/default_pipeline_environment.yml

```diff
@@ -127,6 +127,10 @@ steps:
       - 'opensourceConfiguration'
     toJson: false
     toHtml: false
+  sonarExecute:
+    dockerImage: 'newtmitch/sonar-scanner:3.2.0-alpine'
```

Comment:
does this image work from a permission setup perspective?

Reply:
no, it come with `uid=0(root) gid=0(root) groups=0(root)`

Reply:
if there are files created in the workspace during the run we may see strange permission issues ...

We should have a close look at this.